provider/docker.go

```diff
@@ -270,7 +270,12 @@ func (provider *Docker) getIPAddress(container dockertypes.ContainerJSON) string
 			}
 		}
 	}
-	for _, network := range container.NetworkSettings.Networks {
+	for networkName, network := range container.NetworkSettings.Networks {
```

Comment:
OK :) How to go from `pretty sure` to `sure`? ^^
What could be unreliable using `container.NetworkSettings.Networks`?